### PR TITLE
added upper-case-solution

### LINKED ISSUE #8269: Better explanation for `name`.github.io
### What article on docs.github.com is affected?

<https://github.com/github/docs/blob/main/data/reusables/pages/create-repo-name.md>

### What part(s) of the article would you like to see updated?

Just an addition what to do if org or user name contains uppercase-letters.

### Additional information

If one has uppercase characters in username, it is needed to use lowercase for repo name.

I'll open a merge request to this issue the next minutes.

### PR DESCRIPTION
**Pull request by Barry1**
### Why:

Closes #8269 

### What's being changed:

Just a small additional text that informs users, that they need to select the lowercase variant of `user` or `organization` when creating the repo.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@Barry1 Thanks so much for opening a PR! I'll get this triaged for review 👀

File: data/reusables/pages/create-repo-name.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-1. Type a name for your repository and an optional description. If you're creating a user or organization site, your repository must be named `<user>.github.io` or `<organization>.github.io`. For more information, see "[About {% data variables.product.prodname_pages %}](/articles/about-github-pages#types-of-github-pages-sites)."
+1. Type a name for your repository and an optional description. If you're creating a user or organization site, your repository must be named `<user>.github.io` or `<organization>.github.io`. If `<user>` or `<organization>` contains uppercase letters, you need to lowercase them.
```
Comment (lecoursen) [inline data/reusables/pages/create-repo-name.md:1]: We avoid pronouns whenever possible, as they can be ambiguous.

```suggestion
1. Type a name for your repository and an optional description. If you're creating a user or organization site, your repository must be named `<user>.github.io` or `<organization>.github.io`. If `<user>` or `<organization>` contains uppercase letters, you must lowercase the letters.
```

**Review by lecoursen — COMMENTED** [review]
Looks great, thanks for this! I'll apply my suggestion and merge now. :zap:

**Review by hamedtayeban — APPROVED** [review]

**Review by lecoursen — APPROVED** [review]

**Comment by Barry1** [comment]
> Looks great, thanks for this! I'll apply my suggestion and merge now. ⚡

Thank you for your update. Best regards

**Pull request merged**